### ISSUE TITLE
Add content for "key features" tiles

### ISSUE THREAD

**Issue by rgommers** [numpy/numpy.org#308]
Reported by @asmeurer: 
- One minor comment I have on the site is that the animations on the boxes are a little annoying, and also confusing since the boxes aren't actually links.
- Also the boxes in the bottom apparently are links, but they aren't clickable unless you click on the text.

(Ralf) the first point is about the "key feature" tiles and the warm black underline on hover, the second point about the case study tiles.

**Comment by joelachance** [comment]
I'll +1 to the fact that key features don't do anything. It would be helpful to link these things to additional content throughout the site (what might these links be?)

As for the second, I just checked, and the entire box is clickable, except for a very small part above the blue line. I've PR'ed a fix here: https://github.com/numpy/numpy.org/pull/310

**Comment by rgommers** [comment]
> I'll +1 to the fact that key features don't do anything. It would be helpful to link these things to additional content throughout the site (what might these links be?)

Good point, we could add new content for it. E.g. having the performance box link to some nice benchmarks. Each topic can have a nice single-page summary. Still a bit of work ....

**Comment by InessaPawson** [comment]
The homepage has changed quite a bit since this issue was opened. The concerns raised have been addressed by the current site implementation. Closing the issue as resolved.
